resources/views/home.blade.php

```diff
@@ -4,20 +4,31 @@
 <div class="container">
     <div class="row justify-content-center">
         <div class="col-md-8">
-            <div class="card">
-                <div class="card-header">Dashboard</div>
-
-                <div class="card-body">
-                    @if (session('status'))
-                        <div class="alert alert-success" role="alert">
-                            {{ session('status') }}
-                        </div>
-                    @endif
-
-                    You are logged in!
+            @if (session('status'))
+            <div>
+                新規会員登録またはログインしてください
+            </div>
+            @else
+            @foreach($groups as $group)
+            <div class="card w-25 h-100 mt-1 mr-1 d-inline-block" style="height:170px !important;">
+                <div class="card-header  text-white bg-dark" style="max-height:50px; overflow: hidden;   text-overflow: ellipsis; white-space: nowrap;">{{$group->name}}</div>
+                <div class="card-body" style="max-height:200px; max-width:200px;">
+                    <div style="display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 2;
+    overflow: hidden;">
+                        {{$group->information}}
```

Comment:
これで1行でも2行でもずれないかと！

<img width="680" alt=" 2021-10-09 20 21 33" src="https://user-images.githubusercontent.com/39236040/136656177-e37ca292-e07a-4b87-bd5a-d6fdc686dac5.png">

```suggestion
                        <div style="height: 45px;">{{$group->information}}</div>
```